React InstantSearch Hooks/multi-index-hits/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+import App from './App';
+
+import './index.css';
+
+const container = document.getElementById('root');
+const root = createRoot(container);
+
+root.render(
+  <React.StrictMode>
```

Comment:
I thought we still had timing issues with strict mode, this works?

Reply:
It only breaks routing since [v6.27.0](https://github.com/algolia/react-instantsearch/releases/tag/v6.27.0). But I'm not sure we should enable it by default until we officially support it.